src/pages/blog/PTG_Summary_April_2024.md

```diff
@@ -0,0 +1,129 @@
+
+In addition to the meeting time, session attendees also brought up ideas about how to utilize the meeting time more efficiently.
+
+# Kata Containers on PVM
+Fupan Li gave a presentation to describe [PVM](https://dl.acm.org/doi/abs/10.1145/3600006.3613158), which is a new project developed by Ant Group and Alibaba, and talked about how PVM can serve as a hypervisor for Kata Containers. The presentation also contained benchmarking results and a demo of using the two projects together. PVM is a high-performance guest hypervisor for KVM that is transparent to the host hypervisor and assumes no hardware virtualization support. By using this project, users could avoid performance degradation and other issues that comes from nested virtualization, which is often used when running Kata workloads in public cloud environments, when supported. Using Kata and PVM together could also help further improve the CI and test environments that the community has been using to ensure high code quality.
+
+The PVM community is working with the kernel community to have some required changes accepted. Kata can set up an experimental kernel to start setting up and testing with PVM, but it should not be set as default to avoid maintenance issues and overhead, since both the host and guest kernel needs to be patched.
+
+# Runtime and Hypervisor Updates
+The Go and Rust-based runtimes are a frequent discussion topic within the Kata community. During the PTG session, attendees used the opportunity to discuss runtime as well as hypervisor updates, plans and next steps.
+
+## Runtime-rs Update
+The project had several big improvements since the last update in October, 2023:
+- PCI VFIO Device Support
+  - Including support for basic PCI topology, hot-plug and hot-unplug PCI devices
+- vhost-net support
+- vhost-user-net support
+- and more
+
+“Exclusive” new features (features not available in the Go runtime):
+- FD (File Descriptor) Passthrough support: Dragonball to allow other host programs to pass a FD to the Dragonball process directly. This increases I/O speed and reduces CPU utilization. Adding this feature was part of a student mentorship project.
+- Sandbox API: new interface between containerd and the low-level runtime. More information is available here: https://github.com/kata-containers/kata-containers/pull/7274
+
+Future plans include:
+- TDX support in Dragonball
+- Finalize the Sandbox API
+- Further GPU enhancements to support AI and other workloads
+
+Contributors also raised the question whether or not to switch over to runtime-rs as the default runtime, but no decision was made during the session.
+
+## Hypervisor Updates
```

Comment:
Sorry, I probably should have picked up on this in the etherpad review, but I'm not sure if the current formatting makes it clear that the hypervisor updates are specific to working on getting those hypervisors supported by runtime-rs? Maybe the overall section heading should be `# Runtime and Hypervisor Updates` -> `# Runtime Updates` and this heading should be `## Hypervisor Updates` -> `### Runtime-rs Hypervisor Support Updates`?